test/V4SwapRouter.permit2.t.sol

```diff
@@ -202,11 +203,11 @@ contract V4SwapRouterPermit2Test is SwapRouterFixtures, DeployPermit2, PermitSig
             nonce: 0,
             deadline: block.timestamp + 100
         });
-        bytes memory signature = getPermitTransferToSignature(permit, alicePK, address(manager));
+        bytes memory signature = getPermitTransferToSignature(permit, alicePK, address(router));
 
         bytes memory swapCalldata = abi.encode(
             BaseData({
-                payer: address(router),
+                payer: alice,
```

Comment:
nice was confused why it wasnt behaving as expected, but these changes makes sense